_posts/2023-07-06-good-intentions-gone-bad.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Good intentions gone bad"
+summary: "Learnings on accessibility and how while we learn, we make mistakes and sometimes we make things worse"
+date: 2023-07-06T19:15:00
+meetup: https://www.meetup.com/async-web-tech-meetup/events/293071307/
+speakers:
+  - name: Eva Ferreira
+    link: https://twitter.com/evaferreira92
+sponsors:
+image:
+  url:
+  title:
+  link:
+tags:
+  - accessibility
+  - learning
+  - mistakes
+venue:
+  name: Spaces
+  link: https://www.spacesworks.com/brighton/trafalgar-place/
+  location: https://g.page/Mocatta-House-4437?share
+  address: Mocatta House, Trafalgar Pl, BN1 4DU, Brighton
+  latlong: 50.8294399,-0.1443907
+layout: event.html
+collection: events
+---
+
+Details TBD
```

Comment:
```suggestion
We hope for our web applications to be as beautiful, performant, and accessible as possible; but sometimes... Things get in the way and the outcome is not as good as we hoped. In this talk, I will walk you through a few of my failed attempts at developing accessible web applications. We'll go through different accessibility assumptions and myths that I wrongly believed to be true and how to avoid them by properly testing for accessibility.
```